src/igvfd/tests/fixtures/schemas/analysis_set.py

```diff
@@ -115,3 +115,17 @@ def analysis_set_v7(analysis_set_base):
         'publication_identifiers': ['doi:10.1016/j.molcel.2021.05.020']
     })
     return item
+
+
+@pytest.fixture
```

Comment:
For all your fixtures can you put them above the upgrade fixtures for consistency & style sake?